### PR TITLE
Show connection errors

### PR DESCRIPTION
**Pull request by kamil-stasiak**
- [x] Update `package.json` before merge

### CODE REVIEW

**Review by bblaszkow06 — APPROVED** [review]
LGTM, can be merged after: https://github.com/jellyfish-dev/membrane-webrtc-js/pull/24#pullrequestreview-1282713261

File: assets/src/features/shared/context/ToastContext.tsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ export const ToastProvder = ({ children }: { children?: ReactNode }) => {
       if (toastExists) return;
 
       setToasts((prev) => [...prev, newToast]);
+      if (newToast.timeout === "INFINITY") return;
+
       const timer = setTimeout(() => {
         removeToast(newToast.id);
         clearTimeout(timer);
```
[the remark this answers is not in the document]
Comment (kamil-stasiak) [reply assets/src/features/shared/context/ToastContext.tsx]: 1. You are right, `time` and `setTimeout` were redundant. I've removed them. Thanks! 👍 
2. For me it's easier to read code without if statements, so I'm always trying to change them to guard clauses like this one `if (newToast.timeout === "INFINITY") return;`. I would prefer to stick with my implementation if you don't mind 😅

File: assets/src/pages/room/hooks/useMembraneClient.tsx [hunk 2]
```diff
@@ -127,9 +135,7 @@ export const useMembraneClient = (
         setWebrtc(webrtc);
       })
       .receive("error", (response) => {
-        setErrorMessage("Connecting error");
-        console.error("Received error status");
-        console.error(response);
+        handleError(`Connecting error occurred`);
```
Comment (mickel8) [inline assets/src/pages/room/hooks/useMembraneClient.tsx:138]: Would be nice to give a little bit more descriptive error. E.g. `Couldn't establish signaling connection" or "Failed to establish WS connection"

File: assets/src/pages/room/hooks/useMembraneClient.tsx [hunk 3]
```diff
@@ -103,6 +109,9 @@ export const useMembraneClient = (
         onTrackUpdated: (ctx: TrackContext) => {
           api.setMetadata(ctx.peer.id, ctx.trackId, ctx.metadata);
         },
+        onJoinError: (metadata) => {
+          handleError(`Join error occurred`);
```
Comment (mickel8) [inline assets/src/pages/room/hooks/useMembraneClient.tsx:113]: I would favor

```suggestion
          handleError(`Failed to join the room'`);
```

**Review by mickel8 — APPROVED** [review]

**Pull request merged**